opm/output/data/Wells.hpp

```diff
@@ -255,6 +255,9 @@ namespace Opm {
         double thp;
         double temperature;
         int control;
+
+        ::Opm::Well::Status dynamicStatus { Opm::Well::Status::OPEN };
```

Comment:
The rest of this file is in `namespae Opm { ... }` - is the `::Opm` construction necessary?

Reply:
> is the `::Opm` construction necessary?

Yes.  Writing `Well::Status` here will be treated as a reference to the non-existent type `Opm::data::Well::Status`.